src/client/src/Components/LabelValue/index.tsx

```diff
@@ -7,8 +7,9 @@ interface LabelValueProps {
     label: string;
     value?: string | number;
     isParagraph?: boolean;
+    link?: string;
 }
-function LabelValue({ label, value, isParagraph = false }: LabelValueProps) {
+function LabelValue({ label, value, isParagraph = false, link = undefined }: LabelValueProps) {
```

Comment:
I don't think you need to do link = undefined. Just get the link. If the user doesn't provide it, it will be undefined anyways.

Reply:
Done.